bind/files/debian/named.conf.logging

```diff
@@ -0,0 +1,120 @@
+                print-time yes;
+                print-category yes;
+                print-severity yes;
+                severity info;
+        };
+        channel auth_servers_log {
+                file "{{ map.log_dir }}/auth_servers" versions {{salt['pillar.get']('bind:config:use_extensive_logging:versions', '3')}} size {{salt['pillar.get']('bind:config:use_extensive_logging:size', '20m')}};
+                print-time yes;
+                print-category yes;
+                print-severity yes;
+                severity info;
+        };
+        channel dnssec_log {
+                file "{{ map.log_dir }}/dnssec" versions {{salt['pillar.get']('bind:config:use_extensive_logging:versions', '3')}} size {{salt['pillar.get']('bind:config:use_extensive_logging:size', '20m')}};
+                print-time yes;
+                print-category yes;
+                print-severity yes;
+                severity info;
+        };
+        channel zone_transfers_log {
+                file "{{ map.log_dir }}/zone_transfers" versions {{salt['pillar.get']('bind:config:use_extensive_logging:versions', '3')}} size {{salt['pillar.get']('bind:config:use_extensive_logging:size', '20m')}};
+                print-time yes;
+                print-category yes;
+                print-severity yes;
+                severity info;
+        };
+        channel ddns_log {
+                file "{{ map.log_dir }}/ddns" versions {{salt['pillar.get']('bind:config:use_extensive_logging:versions', '3')}} size {{salt['pillar.get']('bind:config:use_extensive_logging:size', '20m')}};
+                print-time yes;
+                print-category yes;
+                print-severity yes;
+                severity info;
+        };
+        channel client_security_log {
+                file "{{ map.log_dir }}/client_security" versions {{salt['pillar.get']('bind:config:use_extensive_logging:versions', '3')}} size {{salt['pillar.get']('bind:config:use_extensive_logging:size', '20m')}};
+                print-time yes;
+                print-category yes;
+                print-severity yes;
+                severity info;
+        };
+        channel rate_limiting_log {
+                file "{{ map.log_dir }}/rate_limiting" versions {{salt['pillar.get']('bind:config:use_extensive_logging:versions', '3')}} size {{salt['pillar.get']('bind:config:use_extensive_logging:size', '20m')}};
+                print-time yes;
+                print-category yes;
+                print-severity yes;
+                severity info;
+        };
+        channel rpz_log {
+                file "{{ map.log_dir }}/rpz" versions {{salt['pillar.get']('bind:config:use_extensive_logging:versions', '3')}} size {{salt['pillar.get']('bind:config:use_extensive_logging:size', '20m')}};
+                print-time yes;
+                print-category yes;
+                print-severity yes;
+                severity info;
+        };
```

Comment:
How about turning all this block into a map, like (just c&p here, to make an example of what I'm thinking)
```yaml
bind:
  config:
    use_extensive_logging:
       rate_limiting_log:
         file: "{{ map.log_dir }}/rate_limiting" versions {{salt['pillar.get']('bind:config:use_extensive_logging:versions', '3')}} size {{salt['pillar.get']('bind:config:use_extensive_logging:size', '20m')}}
         print-time: yes
         print-category: yes
         print-severity: yes
         severity: info
      rpz_log:
        file: {{ map.log_dir }}/rpz" versions {{salt['pillar.get']('bind:config:use_extensive_logging:versions', '3')}} size {{salt['pillar.get']('bind:config:use_extensive_logging:size', '20m')}}
        print-time: yes
        print-category: yes
        print-severity: yes
        severity: info
...
```
And then iterate over it? That'd make the config of these parameters more granular and would make easier to add/change things to these paramteres. What do you think?

Reply:
Ok I'll do that and enhance it in this way. 
Just only wondering if in pillar user should define such a long value for file.
Also we can move setting for number of files and size into specific channel 

I'll make changes after weekend. Thanks for review

```yaml
bind:
  config:
    use_extensive_logging:
       rpz_log:
        file: rpz
        print-time: yes
        print-category: yes
        print-severity: yes
        severity: info
        versions: 3
        size: 20m
```

this will